doc/source/reference/arrays.datetime.rst

```diff
@@ -393,4 +393,4 @@ As a corollary to this change, we no longer prohibit casting between datetimes
 with date units and datetimes with timeunits. With timezone naive datetimes,
 the rule for casting from dates to times is no longer ambiguous.
 
-.. _pandas: http://pandas.pydata.org
\ No newline at end of file
+.. _pandas: http://pandas.pydata.org
```

Comment:
What changed? This redirects to  https://pandas.pydata.org.

Reply:
nothing changed. It is an artifact of the way `git diff` deals with files with no line ending